ndex2/client.py

```diff
@@ -354,6 +385,21 @@ def save_new_network(self, cx, visibility=None):
     def save_cx_stream_as_new_network(self, cx_stream, visibility=None):
         """
         Create a new network from a CX stream.
+	
+	.. code-block:: python
+	
+		from ndex2.nice_cx_network import NiceCXNetwork
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		node_name = 'INSERT NODE NAME HERE'
+		network_uuid = 'INSERT NETWORK UUID HERE'
+		query_result_cx_stream = my_ndex.get_neighborhood_as_cx_stream(network_uuid, node_name)
+		#visibility is either PUBLIC or PRIVATE
+		my_ndex.save_cx_stream_as_new_network(query_result_cx_stream, visibility=PUBLIC)
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client